framework/aster-frame/src/sync/wait.rs

```diff
@@ -85,35 +86,37 @@ impl WaitQueue {
         }
     }
 
-    /// Wake up one waiting thread.
-    pub fn wake_one(&self) {
+    /// Wakes up at most one waiting thread and returns whether some thread has been woken up.
```

Comment:
A useless implementation that wakes up no waiting thread at all complies with the specification of "waking up at most one waiting thread", so such a description is inaccurate. How about this one?

```rust
/// Wakes up one waiting thread, if there is one at the point of time when this method is called,
/// returning whether such a thread is woken up.
```

Reply:
Makes sense. I've changed accordingly.